android/pytorch_android/src/main/cpp/pytorch_jni.cpp

```diff
@@ -10,9 +10,11 @@
 
 namespace pytorch_jni {
 
-constexpr static int kTensorTypeCodeByte = 1;
-constexpr static int kTensorTypeCodeInt32 = 2;
-constexpr static int kTensorTypeCodeFloat32 = 3;
+constexpr static int kTensorDTypeByte = 1;
+constexpr static int kTensorDTypeInt32 = 2;
+constexpr static int kTensorDTypeFloat32 = 3;
+constexpr static int kTensorDTypeLong64 = 4;
+constexpr static int kTensorDTypeDouble64 = 5;
```

Comment:
is this standard naming in java?  It seems repetitive (is a double ever not 64 bits?).  In python we call this "float64" (and the one above int64).